### PR TITLE
Upload metadata to DynamoDB

### PR DESCRIPTION
**Pull request by bbondy**

### CODE REVIEW

File: scripts/uploadComponent.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ commander
   .option('-d, --crx-directory <dir>', 'directory containing multiple crx files to upload')
   .option('-f, --crx-file <file>', 'crx file to upload', 'extension.crx')
   .option('-t, --type <type>', 'component extension type', /^(ad-block-updater|https-everywhere-updater|tracking-protection-updater)$/i, 'ad-block-updater')
+  .option('-e, --endpoint <endpoint>', 'DynamoDB endpoint to connect to', 'https://dynamodb.amazonaws.com')// If setup locally, use http://localhost:8000
```
[the remark this answers is not in the document]
Comment (bbondy) [reply scripts/uploadComponent.js]: yep actually I was just about to commit for tor and themes

**Comment by bbondy** [comment]
@emerick updated.

**Comment by bbondy** [comment]
I'll probably copy in the release-tools update extension code and just make a clean break from that repo.  Would you be ok with that @emerick ? 
I can then change some hard coded things to work with our setup.

**Comment by emerick** [comment]
@bbondy Absolutely, one fewer moving parts.

**Pull request merged**